wren-ui/src/apollo/server/adaptors/wrenAIAdaptor.ts

```diff
@@ -282,12 +278,13 @@ export class WrenAIAdaptor implements IWrenAIAdaptor {
         `${this.wrenAIBaseEndpoint}/v1/semantics-preparations/${deployId}/status`,
       );
       if (res.data.error) {
-        logger.debug(`deploy error: ${res.data.error}`);
+        // passing AI response error string to catch block
+        throw res.data.error;
```

Comment:
If `res.data.error` is not an error object. Could we wrap it with `Error` ? Maybe we can use our `Errors.create` function ?

Reply:
Maybe wrap it with `Error`? 
The error will be retrieved in the response body as the structure in the image from the deploy API.
<img width="614" alt="image" src="https://github.com/Canner/WrenAI/assets/9657305/711c43a5-b153-46fb-8d5d-ca0cc14dd751">

Using `Errors.create` function currently seems unnecessary here.

Reply:
sure